### PR TITLE
Add an overview for project features

### PR DESCRIPTION
**Pull request by justinbarclay**

### CODE REVIEW

**Comment by justinbarclay** [comment]
This is ready for some feedback

**Review by kekekoko — COMMENTED** [review]
Nice! I left some comments about what I think could be improved.

I think we should also add more information about _how_ and _why_ people should use projects.
Things like
- how do I decide which apps go into which project, do we have any best practices for that?
- how do I fill out the project details? Do we have any articles to link to about, for example how to decide which cloud provider to target? 
...

File: pages/user guides/assessment_projects.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Insights
+
+Finally, after filling in all the information about your project and adding some applications to it, you can get a birds eye view of your project by clicking on the green projects insights button.
+
+{% include image.html file="projects_8.png" %}
+
+This will lead you to our insights page where you can get an overview of your financial highlights, assessment overview, migration ROI, application resource utilization, security vulnerabilities, and technology breakdown
+
+{% include image.html file="projects_9.jpeg" %}
```
[the remark this answers is not in the document]
Comment (justinbarclay) [reply pages/user guides/assessment_projects.md:67]: Yeah this is definitely filler content until we hone in on what we want or find a plugin to do a pop out

File: pages/user guides/assessment_projects.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+{% include image.html file="projects_menu.png" %}
+
+
+## Creating a project
+
+When you first visit https://\<your-subdomain\>.tidalmg.com/#/projects, it looks pretty barren.
+
+{% include image.html file="projects_1.png" %}
+
+However, _you_ can fix that! You can plant the seeds that will lead your orchard of projects to grow into a beautiful forest that will reach the clouds. To start, to plant your first seedling, you will need to click the `Create new project` button in the `Add Project` section found at the top of the page.
```
[the remark this answers is not in the document]
Comment (justinbarclay) [reply pages/user guides/assessment_projects.md:28]: I agree in that there shouldn't be too many jokes in documentation but managers are human and sometimes can have a sense of humour too 😅 

IMO: carefully sprinkling humor in technical documentation improves readability/engagement.

I'm going to keep this _one_ joke but cut the fat by removing any others that I use to build on this theme (IIRC there was one or two)

File: pages/user guides/assessment_projects.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+{% include image.html file="projects_2.png" %}
+
+You will then be brought to a new page that allows you to set out some goals for your new project. This page includes settings for your timelines and financials, target cloud providers, the primary focus of your project, the project sponsor, and the guiding vision for the project.
+
+{% include image.html file="projects_3.png" %}
+
+It's ok if you don't know everything about your burgeoning forest just yet, you can always come back to this page and adjust the details as things evolve.
```
[the remark this answers is not in the document]
Comment (justinbarclay) [reply pages/user guides/assessment_projects.md:36]: I agree forest shouldn't be a theme throughout the doc

**Comment by justinbarclay** [comment]
> * how do I decide which apps go into which project, do we have any best practices for that?
> * how do I fill out the project details? Do we have any articles to link to about, for example how to decide which cloud provider to target?
>   ...

This is a good point! What do you think the answers are? :)

File: pages/user guides/assessment_projects.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+{% include image.html file="projects_6.png" %}
+
+Additionally, you can view the project by the transition types you have chosen for your applications.
+
+{% include image.html file="projects_7.png" %}
+
+## Insights
+
+Finally, after filling in all the information about your project and adding some applications to it, you can get a birds eye view of your project by clicking on the green projects insights button.
```
[the remark this answers is not in the document]
Comment (justinbarclay) [reply pages/user guides/assessment_projects.md:61]: I like Finally here, for me it's a capstone of all the work they've done and how they are now rewarded with getting personalized visualizations on a per project basis.

More of an "Ahhh, finally 😸"

File: pages/user guides/assessment_projects.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+summary: "A overview on the features for Assessment Projects."
+sidebar: main_sidebar
+folder: userguides
+permalink: assessment-projects.html
+---
+
+
+# Assessment Projects
+
+During your migration to the cloud, you might find yourself in charge of dozens or hundreds of applications. At Tidal Migrations, we have found it invaluable to segment our applications into groups. Tidal's Project feature enables users to group applications by a common set of goals or timelines.
```
[the remark this answers is not in the document]
Comment (justinbarclay) [reply pages/user guides/assessment_projects.md]: updated

File: pages/user guides/assessment_projects.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (justinbarclay) [reply pages/user guides/assessment_projects.md]: zoomed in a bit

**Comment by justinbarclay** [comment]
@prees1 I would appreciate your feedback here to make sure we're going down the right track.

**Comment by prees1** [comment]
> @prees1 I would appreciate your feedback here to make sure we're going down the right track.

of course. I gave it a full read and I think it's exactly on track. explains what problem it helps solve and how to do it. 👍

**Pull request merged**